module2/lessons/intro_to_mvc.md

```diff
@@ -14,18 +14,20 @@ title: Introduction to MVC
 - MVC
 - Model
 - View
+- Serializer
 - Controller
 - DSL (Domain Specific Language)
 
 ## Warmup
 
 - Open the Task Manager app you completed earlier in the inning. If you need a fresh copy of that application for any reason, a complete version can be found [here](https://github.com/turingschool-examples/task-manager-7-complete).
-- In your own words, what are the primary responsibilities of the models, views, and controllers in Task Manager?
+<!-- TODO: Will need a complete version of this to link here -->
+- In your own words, what are the primary responsibilities of the models, serializers, and controllers in Task Manager?
 
 ## Overview
 
 - **Models** - Interact with the database. Holds other methods related to a particular resource (e.g. a `task`)
-- **Views** - Templates for pages that we will display to our user. Frequently contain placeholders for data, making them dynamic.
+- **Views** - Presents an application's data to the user. In Task Manager, this was the JSON rendered in the controller with the use of Serializers. In other applications this might look like an HTML template.
```

Comment:
Definitely agree that we should still talk about View as a component of MVC even if it's not a Rails view